yara-sys/build.rs

```diff
@@ -3,13 +3,19 @@
 fn main() {
     build::build_and_link();
     bindings::add_bindings();
+    println!("cargo:rustc-link-lib=dylib=ssl");
+    println!("cargo:rustc-link-lib=dylib=crypto");
```

Comment:
I'm not sure about this.
Also, why do we have to specify this at this place? Shouldn't libyara link the correct libraries?

Reply:
Yara default enable `pe` module witch is depends on https://github.com/VirusTotal/yara/blob/master/libyara/crypto.h which is depends on crypto

Reply:
When link rust-yara with libyara we need this libraries

Reply:
@Hugal31  I add check `openssl`